var/spack/repos/builtin/packages/py-boom/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBoom(PythonPackage):
```

Comment:
I'm a little concerned about the package name. There is already a `boom` package on PyPI that is not the same as this one. Do they have a PyPI release?

Reply:
I found that:
https://pypi.org/project/boom2/

boom2 is just a `hello world` project and have no description on `PyPI` website, 
It looks no update for that PyPI package.

Reply:
If they haven't released on PyPI, we can give them any name that isn't already taken on PyPI, like `py-boom-boot` or `py-boot-manager`

Reply:
So how about `py-boom-boot-manager`, that may descript the `py-boom` feature and reduce conflict rate on `PyPI`.

Reply:
That's fine with me. If the devs don't like the name, they can always apply for a PyPI account.